ncm-dirperm/src/main/perl/dirperm.pm

```diff
@@ -189,6 +211,40 @@ sub process_path {
 				return 0;
 			}
 		} elsif ($type eq "d") {
+            # Should we check if the needed mount is actually mounted?
+            # (i.e. don't build path on / (root) if the mount point is not available)
+            if ($pathentry->{checkmount}) {
+                # Which mountpoint is path built on?
+                my $path_mountpoint = undef;
+
+                foreach my $mntpt (@configured_mounts) {
+                    if ($path =~ /^$mntpt\//) {
```

Comment:
how certain are we that the entries in `configured_mounts` have no trailing slash?